src/tools/clippy/tests/ui/match_wild_err_arm.stderr

```diff
@@ -1,3 +1,12 @@
+error: the feature `exclusive_range_pattern` has been stable since 1.79.0-dev and no longer requires an attribute to enable
+  --> tests/ui/match_wild_err_arm.rs:1:12
+   |
+LL | #![feature(exclusive_range_pattern)]
```

Comment:
This test still has the `#![feature]` attribute, can that be removed or is there some reason for this?

Reply:
No reason, just somehow didn't fail on my computer when testing.